modules/admin_manual/pages/installation/deployment_recommendations.adoc

```diff
@@ -460,4 +464,4 @@ See http://galeracluster.com/documentation-webpages/[the Galera Cluster WSREP do
 
 * http://www.severalnines.com/blog/become-mysql-dba-blog-series-database-high-availability[Database High Availability]
 * http://blog.bitnami.com/2014/06/performance-enhacements-for-apache-and.html[Performance enhancements for Apache and PHP]
-* https://www.digitalocean.com/community/tutorials/how-to-set-up-a-redis-server-as-a-session-handler-for-php-on-ubuntu-14-04[How to Set Up a Redis Server as a Session Handler for PHP on Ubuntu 14.04]
+* https://ubuntuhowtoo.blogspot.com/2019/10/linux-ubuntu-1804-use-redis-for-cache.html[How to Set Up a Redis Server as a Session Handler for PHP on Ubuntu 14.04]
```

Comment:
`[How to Set Up a Redis Server as a Session Handler for PHP on Ubuntu 14.04]` 
-->
`[How to Set Up a Redis Server as a Session Handler for PHP on Ubuntu 18.04]`